pwa/src/components/dashboardCard/DashboardCard.module.css

```diff
@@ -66,7 +90,16 @@
   display: flex;
   gap: var(--gateway-ui-size-md);
 }
-
+.buttonContainer {
+  display: flex;
+  justify-content: space-evenly;
+  gap: 15px;
+  align-items: flex-end;
+}
+.buttonContainer Button {
+  display: flex;
+  flex-grow: 4;
```

Comment:
Unsure what `4` would do here right now.

I think using `flex: 1;` would be better suitable.